financial-connections/res/values/strings.xml

```diff
@@ -31,34 +31,31 @@
     <!-- Account picker -->
     <string name="stripe_account_picker_singleselect_account">Select an account</string>
     <string name="stripe_account_picker_multiselect_account">Select accounts</string>
-    <plurals name="stripe_account_picker_cta_link">
-        <item quantity="one">Connect account</item>
-        <item quantity="other">Connect accounts</item>
-    </plurals>
+
+    <string name="stripe_account_picker_cta_link_singular">Connect account</string>
+    <string name="stripe_account_picker_cta_link_plural">Connect accounts</string>
```

Comment:
Let's track the work to extend the script to handle plurals. I heard that OCS might take on script improvements soon 🤞

Reply:
https://jira.corp.stripe.com/browse/LINK_MOBILE-184